docs/source/usage/plugins.rst

```diff
@@ -88,3 +88,42 @@ If you would like to help in developing those classes for a plugin of your choic
 .. [#f5] Deprecated
 .. [#f6] Only runs on the *CUDA* backend (GPU).
 .. [#f7] Multi-Plugin: Can be configured to run multiple times with varying parameters.
+
+.. rubric:: References
+
+.. [Huebl2014]
+        A. Huebl.
+        *Injection Control for Electrons in Laser-Driven Plasma Wakes on the Femtosecond Time Scale*,
+        Diploma Thesis at TU Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Physiker" (2014),
+        `DOI:10.5281/zenodo.15924 <https://doi.org/10.5281/zenodo.15924>`_
+
+.. [Matthes2016]
+        A. Matthes, A. Huebl, R. Widera, S. Grottel, S. Gumhold, and M. Bussmann
+        *In situ, steerable, hardware-independent and data-structure agnostic visualization with ISAAC*,
+        Supercomputing Frontiers and Innovations 3.4, pp. 30-48, (2016),
+        `arXiv:1611.09048 <https://arxiv.org/abs/1611.09048>`_, `DOI:10.14529/jsfi160403 <https://doi.org/10.14529/jsfi160403>`_
+
+.. [Huebl2017]
+        A. Huebl, R. Widera, F. Schmitt, A. Matthes, N. Podhorszki, J.Y. Choi, S. Klasky, and M. Bussmann.
+        *On the Scalability of Data Reduction Techniques in Current and Upcoming HPC Systems from an Application Perspective.*
+        ISC High Performance Workshops 2017, LNCS 10524, pp. 15-29 (2017),
+        `arXiv:1706.00522 <https://arxiv.org/abs/1706.00522>`_, `DOI:10.1007/978-3-319-67630-2_2 <https://doi.org/10.1007/978-3-319-67630-2_2>`_
+
+.. [Pausch2012]
+        R. Pausch.
+        *Electromagnetic Radiation from Relativistic Electrons as Characteristic Signature of their Dynamics*,
+        Diploma Thesis at TU Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Physiker" (2012),
+        https://www.hzdr.de/db/Cms?pOid=38997
+
+.. [Pausch2014]
+        R. Pausch, A. Debus, R. Widera, K. Steiniger, A.Huebl, H. Burau, M. Bussmann, and U. Schramm.
+        *How to test and verify radiation diagnostics simulations within particle-in-cell frameworks*,
+        Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment 740.11, pp. 250-256 (2014)
```

Comment:
There is not extra `.11` after `740` this is just the day of publication. (neither the month nor anything relevant)

Reply:
must have mismatched this with the edition when I copied this. will just remove the 11